src/uct/ib/rc/base/rc_iface.c

```diff
@@ -99,6 +99,13 @@ ucs_config_field_t uct_rc_iface_common_config_table[] = {
    "Otherwise poll TX completions only if no RX completions found.",
    ucs_offsetof(uct_rc_iface_common_config_t, tx.poll_always), UCS_CONFIG_TYPE_BOOL},
 
+  {"CONN_ECE", "0",
+   "config Enhanced Connection Establishment to establish connection.\n"
+   "  0           : use default ECE.\n"
+   "  auto        : use default max ECE.\n"
```

Comment:
0  : Use default ECE
auto : Use maximal supported ECE
otherwise : Set the ECE to the given numeric 32-bit value. This value is used as best-effort, and can be adjusted by the transport implementation.

Reply:
Done